src/css/profile/mobile/common/timepicker.less

```diff
@@ -22,6 +22,28 @@
 		}
 	}
 
+	&[data-format="24"] {
+		.ui-time-picker-container {
+			width: 30%;
+
+			&-hour::after {
+				content: ":";
+				height: 164 * @px_base;
+				width: calc(40% / 3);
```

Comment:
I'm not sure your intention. You wanted `calc` to be done in `less` or `css`?
For `less` will be enough `40% / 3` but in css '~"calc(40% / 3)"'

Reply:
I think it was typo

Reply:
Okay. I will recover it to done it in `less`.